src/frontend/apps/impress/src/features/docs/docs-grid/components/DocsGrid.tsx

```diff
@@ -46,12 +46,11 @@ export const DocsGrid = ({
     void fetchNextPage();
   };
 
-  const title =
-    target === DocDefaultFilter.MY_DOCS
-      ? t('My docs')
-      : target === DocDefaultFilter.SHARED_WITH_ME
-        ? t('Shared with me')
-        : t('All docs');
+  const title = {
+      [DocDefaultFilter.MY_DOCS]: t('My docs'),
+      [DocDefaultFilter.SHARED_WITH_ME]: t('Shared with me'),
+      [DocDefaultFilter.ALL_DOCS]: t('All docs'),
+  }[target]
```

Comment:
Yes the default fallback `t('All docs')` as to be defined as target can be `undefined`:
https://github.com/suitenumerique/docs/blob/003d55020d02d68e3a06cc8ffc7bd1c9ff156d00/src/frontend/apps/impress/src/features/docs/docs-grid/components/DocsGrid.tsx#L16